AK/Buffered.h

```diff
@@ -108,6 +108,9 @@ class Buffered<StreamType, Size, typename EnableIf<IsBaseOf<InputStream, StreamT
         return true;
     }
 
+    size_t buffered() const { return m_buffered; }
+    StreamType& leak_stream() const { return m_stream; }
```

Comment:
The thing is that if you just read from the stream received through this method, in 99% of cases you're gonna brick the buffering behavior. So I'm putting in a warning stating that because the user needs to take care with this function.